src/pal/src/pal_load_file.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "pal_log.h"
+
+char *pal_load_file(const char *filename) {
```

Comment:
I think the name should be `pal_load_file_to_string`

Reply:
Done